tests/app/forms/test_household_relationship_form.py

```diff
@@ -131,14 +130,14 @@ def test_serialise_relationship_answers(self):
 
         expected_answers = [
             {
-                'group_id':"household-relationships",
+                'group_id': "household-relationships",
```

Comment:
do we not check the consistency of `'` and `"`?

Reply:
This should be picked up by linting as part of the PR done by Hamish

Reply:
Running pylint against the `tests` directory appears to not be linting all python files. If you run `find ./tests -type f -name "*.py" | xargs pipenv run pylint` a lot of lint errors (incluing ones for quotes) are generated.